hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeExtension.java

```diff
@@ -283,63 +288,85 @@ private void printNodeInfoInternal(BuildInfo buildInfo, String build) {
         boolean isMulticastJoin = node.shouldUseMulticastJoiner(getActiveMemberNetworkConfig(config).getJoin());
-        addFeatureInfo(sb, "Member multicast discovery/join method disabled", !isMulticastJoin);
-        addFeatureInfo(sb, "Advanced networking used - Sockets separated", config.getAdvancedNetworkConfig().isEnabled());
+        addSecurityFeatureCheck(sb, "Disable member multicast discovery/join method", !isMulticastJoin);
+
         AdvancedNetworkConfig advancedNetworkConfig = config.getAdvancedNetworkConfig();
-        addFeatureInfo(sb, "Server sockets bound to a single network interface",
-                !node.getProperties().getBoolean(ClusterProperty.SOCKET_SERVER_BIND_ANY));
-        AuditlogConfig auditlogConfig = config.getAuditlogConfig();
+        addSecurityFeatureCheck(sb, "Use advanced networking, separate client and member sockets",
+                advancedNetworkConfig.isEnabled());
+        boolean bindAny = node.getProperties().getBoolean(SOCKET_SERVER_BIND_ANY);
+        addSecurityFeatureCheck(sb,
+                "Bind Server sockets to a single network interface (disable " + SOCKET_SERVER_BIND_ANY.getName() + ")",
+                !bindAny);
+        StringBuilder tlsSb = new StringBuilder();
+        boolean tlsUsed = true;
+        if (advancedNetworkConfig.isEnabled()) {
+            for (Map.Entry<EndpointQualifier, EndpointConfig> e : advancedNetworkConfig.getEndpointConfigs().entrySet()) {
+                tlsUsed = addAdvNetworkTlsInfo(tlsSb, e.getKey(), e.getValue().getSSLConfig()) && tlsUsed;
+            }
+        } else {
+            SSLConfig sslConfig = config.getNetworkConfig().getSSLConfig();
+            tlsUsed = addSecurityFeatureCheck(tlsSb, "Use TLS communication protection (Enterprise)",
+                    sslConfig != null && sslConfig.isEnabled());
+        }
         boolean jetEnabled = config.getJetConfig().isEnabled();
-        addFeatureInfo(sb, "Jet disabled", !jetEnabled);
         if (jetEnabled) {
-            addFeatureInfo(sb, "Jet resource upload disabled", !config.getJetConfig().isResourceUploadEnabled());
+            boolean trustedEnv = tlsUsed || !bindAny;
+            addSecurityFeatureCheck(sb, "Use Jet in trusted environments only (single network interface and/or TLS enabled)",
+                    trustedEnv);
+            addSecurityFeatureCheck(sb, "Disable Jet resource upload", !config.getJetConfig().isResourceUploadEnabled());
+        } else {
+            addSecurityFeatureInfo(sb, "Jet is disabled");
         }
-        addFeatureInfo(sb, "User code deployment disallowed", !config.getUserCodeDeploymentConfig().isEnabled());
-        addFeatureInfo(sb, "Scripting in Management Center disabled", !config.getManagementCenterConfig().isScriptingEnabled());
+        addSecurityFeatureCheck(sb, "Disable user code deployment", !config.getUserCodeDeploymentConfig().isEnabled());
```

Comment:
I would also mention this should only be done in trusted environments

Reply:
:+1: